### PR TITLE
Adds configuration option for delay before fading links after clickin…

### PR DESCRIPTION
**Pull request by tyler-m-dunn**
…g "hide".

### CODE REVIEW

File: lib/modules/betteReddit.js [hunk 1]
```diff
@@ -287,7 +293,7 @@ async function hideLink(clickedLink) {
 	} else {
 		$(clickedLink).text('unhide');
 		clickedLink.setAttribute('action', 'unhide');
-		hideTimer = setTimeout(() => parentThing.$thing.fadeOut(300), 5000);
+		hideTimer = setTimeout(() => parentThing.$thing.fadeOut(300), module.options.hideLinkFadeDelay.value);
```
Comment (andytuba) [inline lib/modules/betteReddit.js]: Could you wrap the `...value`  in `parseInt(..., 10)`?

**Comment by tyler-m-dunn** [comment]
Done. Thanks for taking a look so quickly.

File: lib/modules/betteReddit.js [hunk 2]
```diff
@@ -33,7 +33,13 @@ module.options = {
 	fixHideLinks: {
 		type: 'boolean',
 		value: true,
-		description: 'Changes "hide" links to read as "hide" or "unhide" depending on the hide state. Also adds a 5 second delay prior to hiding the link.',
+		description: 'Changes "hide" links to read as "hide" or "unhide" depending on the hide state.',
+	},
+	hideLinkFadeDelay: {
```
Comment (larsjohnsen) [inline lib/modules/betteReddit.js]: Should add `dependsOn: 'fixHideLinks'`

**Comment by tyler-m-dunn** [comment]
Done.

**Comment by erikdesjardins** [comment]
Thanks!

**Pull request merged**